tests/csm/rest/test_csm_load.py

```diff
@@ -319,6 +332,87 @@ def test_44794(self):
         self.log.info("##### Test completed -  %s #####", test_case_name)
 
 
+    @pytest.mark.jmeter
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.tags('TEST-45719')
+    def test_45719(self):
+        """
+        CSM load testing with Get User Capacity while running IOs in parallel
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+
+        self.log.info("Step 1: Send multiple create CSM users with same creds requests parallel")
+        fpath = os.path.join(self.jmx_obj.jmeter_path, self.jmx_obj.test_data_csv)
+        content = []
+        fieldnames = ["role", "user", "pswd"]
+        content.append({fieldnames[0]: "admin",
+                        fieldnames[1]: "admin_test_45719",
+                        fieldnames[2]: CSM_REST_CFG["csm_admin_user"]["password"]})
+        content.append({fieldnames[0]: "manage",
+                        fieldnames[1]: "manage_test_45719",
+                        fieldnames[2]: CSM_REST_CFG["csm_user_manage"]["password"]})
+        content.append({fieldnames[0]: "monitor",
+                        fieldnames[1]: "monitor_test_45719",
+                        fieldnames[2]: CSM_REST_CFG["csm_user_monitor"]["password"]})
+        self.log.info("Test data file path : %s", fpath)
+        self.log.info("Test data content : %s", content)
+        config_utils.write_csv(fpath, fieldnames, content)
+        jmx_file = "CSM_Create_N_CSM_Users.jmx"
+        self.log.info("Running jmx script: %s", jmx_file)
+        result = self.jmx_obj.run_verify_jmx(
+            jmx_file, expect_error_count = self.request_usage - 3,
+            threads=self.request_usage, rampup=1, loop=1)
+        assert result, "Errors reported in the Jmeter execution"
+
+        self.log.info("Step 2: Add user to list to be deleted")
+        self.csm_users_created.append("admin_test_45719")
```

Comment:
fetch this variable from content and not hardcode

Reply:
updated